server/controllers/chartsController.js

```diff
@@ -2,18 +2,23 @@ import { ObjectID } from 'mongodb'
 
 import { collections } from '../utils/mongoCollections'
 
+const studyCountsToPercentage = (studyCount, targetTotal) =>
+  (100 * studyCount) / targetTotal
+
 const postProcessData = (data, studyTotals) => {
   const processedData = {}
-
   Object.entries(data).forEach((entry) => {
     const [key, count] = entry
     const [study, valueLabel, color, studyTarget] = key.split('-')
+    const totals = studyTotals[study].targetTotal || studyTotals[study].count
+    const percent = studyCountsToPercentage(+count, totals)
```

Comment:
🎱 could move the `+` to line 6, that way we don't need to remember to use it in multiple places --  also I think it'd be more readable to use something like `parseInt` or `parseFloat` here, just my 2¢

Reply:
https://github.com/AMP-SCZ/dpdash/pull/125/commits/825202fb4464e2ca01467a863b7d1e14823e6808

Thanks for that.

Reply:
do we need to coerce the second parameter to a number as well?

Reply:
That's happening here. https://github.com/AMP-SCZ/dpdash/blob/8fa23ad39ab6fd14bf85f199f0d909ecd3c962f2/server/controllers/chartsController.js#L72